### ISSUE TITLE
After execuring a function, NodeJS Console takes very long or hangs on `.exit`

### ISSUE THREAD

**Issue by flofrie** [clearmatics/autonity#728]
Trying NodeJS Console against my own Node, as below, the console hangs after typing `.exit`. `<CTRL-C>` helps, but shouldn't be required.

(Entering console, then immediately doing `.exit` seems to hang sometimes, but not always!?)

```
$ ./console ws://51.89.148.237:8546

Welcome to the Autonity node console
modules: admin:1.0 debug:1.0 eth:1.0 miner:1.0 net:1.0 personal:1.0 rpc:1.0 tendermint:1.0 txpool:1.0 
Type web3.<Tab> or autonity.<Tab> to get started
> contract.options.gasPrice=0
0
> await autonity.getWhitelist().call()
[
  'enode://181dd52828614267b2e3fe16e55721ce4ee428a303b89a0cba3343081be540f28a667c9391024718e45ae880088bd8b6578e82d395e43af261d18cedac7f51c3@35.246.21.247:30303',
  'enode://e3b8ea9ddef567225530bcbae68af5d46f59a2b39acc04113165eba2744f6759493027237681f10911d4c12eda729c367f8e64dfd4789c508b7619080bb0861b@35.189.64.207:30303',
  'enode://00c6c1704c103e74a26ad072aa680d82f6c677106db413f0afa41a84b5c3ab3b0827ea1a54511f637350e4e31d8a87fdbab5d918e492d21bea0a399399a9a7b5@34.105.163.137:30303',
  'enode://dffaa985bf36c8e961b9aa7bcdd644f1ad80e07d7977ce8238ac126d4425509d98da8c7f32a3e47e19822bd412ffa705c4488ce49d8b1769b8c81ee7bf102249@35.177.8.113:30308',
  'enode://1bd367bfb421eb4d21f9ace33f9c3c26cd1f6b257cc4a1af640c9af56f338d865c8e5480c7ee74d5881647ef6f71d880104690936b72fdc905886e9594e976d1@35.179.46.181:30309',
  'enode://a7465d99513715ece132504e47867f88bb5e289b8bca0fca118076b5c733d901305db68d1104ab838cf6be270b7bf71e576a44644d02f8576a4d43de8aeba1ab@3.9.98.39:30310',
  'enode://1fb2cdc46bc249ab230753b2a5fa25ef8a24dc61fb8c3489c13dd222182d1d29cf13261bcff5b72ee2c3be9e89a3ec50ace6fe0bd7aec467a8a44af20804d743@51.38.71.12:30303',
  'enode://a8e50b0a66b3b2786340d341dc9ce380d866349f806b14f53a5fa15493562986f3f92e6f9e32c563c879431da42e349af825f89f34a13801b3494bc4ad1066d8@51.89.167.63:30303',
  'enode://c303846cb2e942c929b5aeeb8447c623f9af2a919a7322fdc1751aa663012ed4a8edb5effe634154964d250fb2863cf57bcaa15f2a05fb3df7d128c99c3b4c12@34.105.160.254:30303',
  'enode://e0b03c7bc0ec66f081ca2c1a4c50d87697b1dc44c0213e7527da7e8b213dba4e6670b4a0f8424c110fa34f72f665387a7a00f0f15d0e3298e1783cddf25af50b@51.89.148.237:30303',
  'enode://957d4bab46f12578d43e9e178dfae2bad27e5da956f48cae5897dc21d9d486b1229005d8c3819632329f08414c72a35ac15d295b9ccdc4c4fadbffee8505551f@34.105.218.196:30303',
  'enode://ebf0fb6aa0437ed08a2eaf365774de4e4c5b1a845a60eefa1a102f39d157bb0c4ad9e1c23d5aa6a510cdf8b003033a22455ca2f42c9f11cd2c31efe4ed1c20b3@35.197.221.17:30303',
  'enode://93125e2aa8cc971cd043610a9da17c26bc682a71c096eca4ce7a47c4068f71e8a494f0981023e0ad5a8a4030026ec8fc43fa96930f2630c0e5d9f82e568c1eb3@35.197.198.209:30303',
  'enode://c3626c0e879d4c64f23e1370ae96e62a2e332db55f3b4c28fb02c9c0a7df8cd84d365f465ac67c6095ff4359a7169eb9b62d844e1ab28f9def716afd26269858@51.140.10.245:30303',
  'enode://db915879e5e283a06e61154669cefc10ffc3d056decdd8fe9e9b43abb0a36bb5e3f96e38a0affcd0a57cd8f1efe13e979b2920545d5ec32f0b54a42316ce818c@51.195.149.229:30303'
]
> .exit

```

**Comment by piersy** [comment]
@flofrie Well spotted, thank you for this bug report! I believe this is fixed in #729, could you let me know if that fix is working for you?

**Comment by flofrie** [comment]
@piersy , fix is working.

@yazzaoui , how shall we handle this in terms of the release? Once this patch is merged, shall we tag a v0.7.1 release? Or stick with v0.7.0 for now, and just mention this very minor bug in the user documentation?
